### DISCUSSION TITLE
azure-devops-agent-with-docker-compose/

### DISCUSSION THREAD

**Discussion by feveden** [alkampfergit/personal-blog#17] [Q&A]
https://www.codewrecks.com/post/old/2019/12/azure-devops-agent-with-docker-compose/ 
Hi, can you supply the azure devops pipeline file for the above post?
Thank you.

**Comment by alkampfergit** [comment]
Actually that post simply deal with a way of creating an agent based on docker, then you can run any pipeline on it
